blt/src/Task/Npm/RunScript.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Cgov\Task\Npm;
+
+use Robo\Contract\CommandInterface;
+use Robo\Exception\TaskException;
+use Robo\Task\Npm\Base;
+
+/**
+ * Implementation of the missing 'npm run-script' command..
+ *
+ * @package Cgov\Task\Npm
+ */
+class RunScript extends Base implements CommandInterface {
+
+  /**
+   * {@inheritDoc}
+   */
+  protected $action = 'run-script';
+
+  /**
+   * Set the name of the script to be executed.
+   *
+   * @param mixed $scriptName
+   *   Name of the script to execute.
+   *
+   * @return $this
+   *   Instance of the task
+   */
+  public function scriptName($scriptName) {
```

Comment:
See my note above about how npm run can take additional parameters with the `--` option

Reply:
👍